### ISSUE TITLE
Cannot run with coverage with latest dependencies

### ISSUE THREAD

**Issue by newmana** [zwilias/elm-coverage#34]
I get the following error after I've upgraded my JS dependencies:
```$ yarn run coverage
yarn run v1.22.10
$ yarn run elm-coverage
$ ~/chroma-elm/node_modules/.bin/elm-coverage
[07:18:01.97] Instrumenting sources...
[07:18:03.49] Running tests...
Solving dependencies > Compiling > Starting tests
[07:18:08.16] Ruh roh, tests failed.
[07:18:08.16] Something went wrong:
Error: ~/chroma-elm/.coverage/instrumented/elm-stuff/generated-code/elm-community/elm-test/0.19.1-revision6/elmTestOutput.js:6702
    app.ports.send.subscribe(function(rawData) {
                   ^

TypeError: Cannot read property 'subscribe' of undefined
    at Timeout._onTimeout (~/chroma-elm/.coverage/instrumented/elm-stuff/generated-code/elm-community/elm-test/0.19.1-revision6/elmTestOutput.js:6702:20)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
```

I don't think it's an elm dependency problem, I've "narrowed" it down to the differences in the yarn.lock:
https://github.com/newmana/chroma-elm/compare/671d9f72c08b5a05a950d69d2f7cb946f669f61d..1a4a091ed7f7969710dce775564128a692765295

**Comment by newmana** [comment]
My mistake using the following fork fixed it for me:
https://github.com/marc136/elm-coverage/tree/add_support_for_current_elm-test

**Comment by marc136** [comment]
I also created PR #31 from that fork. I hope it will land here, too.

**Comment by loicknuchel** [comment]
I have the same problem, can't run elm-coverage because of the missing subscribe.
@marc136 did you have a workaround to make it work while your PR is not merged/released ?

**Comment by marc136** [comment]
Yes, @loicknuchel, I just install my elm-coverage fork directly from github.

E.g. `npm install -D github:marc136/elm-coverage`, which will produce this line in the package.json:

```js
{ ...
  "devDependencies": {
    "elm-coverage": "github:marc136/elm-coverage"
  }
}

```

**Comment by loicknuchel** [comment]
Thanks :+1:

**Comment by zwilias** [comment]
Released 0.4.0 which should be compatible with current versions 🎉

**Comment by newmana** [comment]
Thank you so much for fixing this.
